CMakeLists.txt

```diff
@@ -466,7 +466,16 @@ else (WITH_THREADS AND Threads_FOUND)
   set (NO_THREADS 1)
 endif (WITH_THREADS AND Threads_FOUND)
 
-set (TEST_SRC_DIR \"${CMAKE_CURRENT_SOURCE_DIR}\")
+# fopen/open on Cygwin can not handle unix-type paths like /home/.... 
+# therefore we translate TEST_SRC_DIR to windows-path.
+if (CYGWIN)
+  EXECUTE_PROCESS(COMMAND cygpath.exe -m ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
I'm not sure this is really necessary. Is it possible that this happens if one uses a non-Cygwin distributed CMake for Cygwin builds?

Besides that, glog conventions for CMake formatting should be used (lowercase, space between command name and parenthesis).

Reply:
I installed cmake via cygwin package manager. Without this fix TEST_SRC_DIR is defined in config.h as `"/home/roman/glog"`. 
This, however, fails at googletest.h:446 because fopen() can not handle such path style.
With this fix TEST_SRC_DIR  is exported as `"C:/cygwin64/home/roman/glog"`